opentracing-spring-cloud-starter/src/test/java/io/opentracing/contrib/spring/cloud/NoDepsTest.java

```diff
@@ -0,0 +1,30 @@
+package io.opentracing.contrib.spring.cloud;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * @author Pavol Loffay
+ */
+public class NoDepsTest {
+
+    @Test
+    public void testNoFeign() {
+        try {
+            this.getClass().getClassLoader().loadClass("feign.Client");
+            this.getClass().getClassLoader().loadClass("feign.hystrix.HystrixFeign");
+            Assert.fail();
+        } catch (ClassNotFoundException e) {
+        }
+    }
+
+    @Test
+    public void testNoJMS() {
+        try {
+            this.getClass().getClassLoader().loadClass("javax.jms.Message");
```

Comment:
I think these should be in separate try/catch blocks, as `javax.jms.Message` may not be in the classpath, but `org.springframework.jms.core.JmsTemplate` is - but that wouldn't be picked up. Same may apply to the above test, assuming they are in different maven artifacts.

Reply:
I have split the tests